src/components/ChildrenPanel.vue

```diff
@@ -0,0 +1,77 @@
+<script lang="ts">
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
```

Comment:
should be able to remove these if everything is typed correctly.

Reply:
I get one any-warning in line 26: `data = await this.$api.getChildren(this.account);`. Not sure how to fix it